File: Plugins/QuorumPlugins/src/main/java/plugins/quorum/Libraries/Language/Support/TextBuilder.java
package plugins.quorum.Libraries.Language.Support;

public class TextBuilder {
    public java.lang.Object me_ = null;
    StringBuilder builder = new StringBuilder();

    public void Append(String value) {
        builder.append(value);
    }
    public int GetMaxSize() {
        return builder.capacity();
    }
    public void SetMaxSize(int size) {
        builder.ensureCapacity(size);
    }
    public int GetSize() {
        return builder.length();
    }
    public String ToText() {
        return builder.toString();
    }
}

File: Quorum/Library/Standard/Plugins/javascript/Libraries/Language/Support/TextBuilder.js
function plugins_quorum_Libraries_Language_Support_TextBuilder_() {
    this.text = "";

    this.Append$quorum_text = function (value) {
        this.text = this.text + value;
    };

    this.GetMaxSize = function () {
        return this.text.length;
    };

    this.SetMaxSize$quorum_integer = function (value) {
    };

    this.ToText = function () {
        return this.text;
    };
}